### ISSUE TITLE
Error on org-mode

### ISSUE THREAD

**Issue by colawithsauce** [ianpan870102/vscode-default-dark-emacs-theme#37]
```lisp
   `(org-todo                         ((,class (,@(when vscode-dark-plus-box-org-todo (list :box '(:line-width 1 :color ,ms-lightred))) ;; <- HERE
                                                :foreground ,ms-lightred :bold nil))))
```
this snippet doesn't work on emacs-30, master branch. Because `,ms-lightred` doesn't expand as expected.

**Comment by ianyepan** [comment]
Interesting find. I'm using Emacs 29 myself -- probably why I've never run into this issue. Do you know of a workaround for this, or why Emacs 30 doesn't expand variables like previous versions do?

**Comment by colawithsauce** [comment]
yes, indeed. currently I just added this line into my init file:
```emacs-lisp
(setq vscode-dark-plus-box-org-todo nil)
```

**Comment by lukedalessandro** [comment]
> yes, indeed. currently I just added this line into my init file:
> 
> (setq vscode-dark-plus-box-org-todo nil)

Even with this set, I get the error `Invalid face box: :line-width, 1, :color, (\, ms-lightred)` when I try and load org-mode for the first time. Once this fails once, org-mode seems to work okay. Any idea what I'm doing wrong?

**Comment by ianyepan** [comment]
PR #40 and #41 (both merged) fixed the issues presented in this thread. Closing ticket - feel free to reopen if it you still observe this issue - thanks!
